JNWCollectionView/JNWCollectionViewFramework.m

```diff
@@ -1093,6 +1118,20 @@ - (void)moveLeftAndModifySelection:(id)sender {
 	[self selectItemAtIndexPath:toSelect atScrollPosition:JNWCollectionViewScrollPositionNearest animated:YES selectionType:JNWCollectionViewSelectionTypeExtending];
 }
 
+- (void)moveToBeginningOfDocument:(id)sender {
+	NSIndexPath *toSelect = [self firstIndexPath];
+	if (toSelect) {
```

Comment:
There is already a nil check in `selectItemAtIndexPath:atScrollPosition:animated`.